components/datetime/examples/work_log.rs

```diff
@@ -34,6 +37,8 @@ fn print(_input: &str, _value: Option<usize>) {
 
 #[no_mangle]
 fn main(_argc: isize, _argv: *const *const u8) -> isize {
+    main_setup!();
```

Comment:
nit; could you instead not use `use *;` and do:

```rust
icu_benchmark_macros::static_setup!();
```
```rust
icu_benchmark_macros::main_setup!();
```

This may seem like more boilerplate, but it makes it very visually clean that those two lines are benchmark related macros and can be skipped when interpreting the code.

Reply:
I agree, this is nicer.